hugo/layouts/papers/single.html

```diff
@@ -144,9 +144,13 @@ <h5 class="card-title">Abstract</h5>
       <dt>Pages:</dt>
       <dd>{{ with $paper.pages }}{{ . }}{{ end }}</dd>
       <dt>URL:</dt>
-      <dd>{{ with $paper.pdf }}<a href="{{ . }}">{{ . }}</a>{{ end }}</dd>
+      <dd><a href="{{ $paper.url }}">{{ . }}</a></dd>
```

Comment:
That second dot also needs to be replaced by `$paper.url` in this case. :)

Thanks @akoehn for testing & noticing!